Upstream Commit: https://github.com/zynaddsubfx/zynaddsubfx/commit/f384d92486d6b515cb628d0f52008a9e03341d8c
Upstream Commit: https://github.com/zynaddsubfx/zynaddsubfx/commit/70905c96fe7b9ffde19bc4bc05b0dc53a1ed1707

From f384d92486d6b515cb628d0f52008a9e03341d8c Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 19 Apr 2023 21:52:01 -0400
Subject: [PATCH] Bank: Add stdint.h header include

As uint8_t is used, this header should be included.
---
 src/Misc/Bank.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Misc/Bank.h b/src/Misc/Bank.h
index 5120441a5..3f324dd84 100644
--- a/src/Misc/Bank.h
+++ b/src/Misc/Bank.h
@@ -18,6 +18,7 @@
 #include <vector>
 #include "../globals.h"
 #include "Config.h"
+#include <stdint.h>
 
 //entries in a bank
 #define BANK_SIZE 160

From 70905c96fe7b9ffde19bc4bc05b0dc53a1ed1707 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sat, 22 Apr 2023 10:21:09 -0400
Subject: [PATCH] MidiIn: Add stdint.h header include

---
 src/Nio/MidiIn.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Nio/MidiIn.h b/src/Nio/MidiIn.h
index ce0bcfec3..c6b30c61d 100644
--- a/src/Nio/MidiIn.h
+++ b/src/Nio/MidiIn.h
@@ -17,6 +17,7 @@
 #define MIDI_IN_H
 
 #include "Engine.h"
+#include <stdint.h>//uint8_t
 
 namespace zyn {
